example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl

```diff
@@ -18,18 +18,18 @@
 
 :process_1 a sio:SIO_001001 ;
   sio:SIO_000229 :output_1;
-  prov:startedAtTime  "2017-10-18T13:00:00"^^xsd:dateTime .
+  sio:SIO_000680  "2017-10-18T13:00:00"^^xsd:dateTime .
   
 :process_2 a sio:SIO_000006 ;
   sio:SIO_000229 :output_2;
-  prov:wasInformedBy :process_1;
-  prov:startedAtTime  "2015-10-18T13:00:00"^^xsd:dateTime .
+  sio:SIO_000230 :output_1; # has input
```

Comment:
Are you sure that you want to add a comment here? (Also in ShEx)

Reply:
Not sure if we can keep them or remove them. But I removed it for now